### ISSUE TITLE
identical "time" values in ibmqx3 job execution results don't seem correct

### ISSUE THREAD

**Issue by ku-antum** [QISKit/qiskit-api-py#34]
When looking at job-results I noticed that the job's executions on ibmqx3 always have identical "time" values (down to the 14th digit) even when their QASM programs differed in length and complexity.
(Simulations give different "time" values even on the millisecond scale (assuming that the time unit is milliseconds) and even for identical QASM programs per execution.)

Is the "time" value set to the same value for all ibmqx3 executions of a single job or is that a mistake?
Could it be that the value in "time" is the cumulated time taken for all executions of this job together?  
At least the "date" fields varies for executions of a job on ibmqx3, on the scale of seconds even.

For clarification the "time" and "date" fields I'm referring to are:

{"qasms":[{"qasm":"include \"qelib1.inc\";qreg q[16] ...",
           "status":"DONE",
           "executionId":"...",
           "result":{**"date"**:"...",
           "data":{
           ..., **"time"** :100.85948204994202,
           "counts": ...

**Comment by pacomf** [comment]
Yes, now it is not possible. We are working on that right now :). So, in next versions it will be possible :)
